tests/unit/test_generate_payload_fixtures.py

```diff
@@ -35,6 +38,15 @@ def request_payload_dir_fixture():
     return "tests/fixtures/request_payloads"
 
 
+def replace_obj_id(obj: Any, obj_id: ObjectId) -> Any:
+    """
+    Helper function to replace the object ID of the type
+    """
+    params = obj.dict()
+    params["_id"] = obj_id
```

Comment:
If we replace `obj.dict()` to `obj.dict(by_alias=True)`, it should fix the issue without introducing this method.